lua/weapons/weapon_ttt_m16_stn.lua

```diff
@@ -0,0 +1,158 @@
+-- NLL created by https://steamcommunity.com/profiles/76561198813867340
+
+include("autorun/nl_library.lua")
+
+if SERVER then
+    AddCSLuaFile()
+end
+
+DEFINE_BASECLASS("weapon_tttbase")
+
+SWEP.HoldType = "ar2"
+
+if CLIENT then
+    SWEP.PrintName = "M16 (STN)"
+    SWEP.Slot = 6
+
+    SWEP.ViewModelFlip = false
+    SWEP.ViewModelFOV = 64
+
+    SWEP.EquipMenuData = {
+        type = "item_weapon",
+	name = "M16 (STN)",
+        desc = "A Non-Lethal M16.\n\nHeadshots deal 92 stun for 15 seconds.\n\nBodyshots deal 23 stun for 10 seconds."
+    }
+
+    SWEP.Icon = "vgui/ttt/icon_m16"
+    SWEP.IconLetter = "w"
+end
+
+SWEP.Base = "weapon_tttbase"
+
+SWEP.Kind = WEAPON_EQUIP
+SWEP.WeaponID = AMMO_M16
+SWEP.CanBuy = { ROLE_DETECTIVE , ROLE_TRAITOR }
+SWEP.LimitedStock = false
+SWEP.builtin = true
+SWEP.spawnType = WEAPON_TYPE_HEAVY
```

Comment:
This field tells TTT it should spawn on the map by default at the specified spawn node type. If this is meant to only be available in the shop (which I assume it is), this should be left out.

Reply:
When setting the base to M16, set this field to nil instead